iosApp/iosApp/Pages/TripDetails/TripDetailsStopListSplitView.swift

```diff
@@ -23,6 +23,17 @@ struct TripDetailsStopListSplitView: View {
                         TripDetailsStopView(stop: stop, now: now, onTapLink: onTapLink, routeType: routeType)
                     }
                 }
+                .accessibilityElement()
+                .accessibilityAddTraits(/*@START_MENU_TOKEN@*/ .isHeader/*@END_MENU_TOKEN@*/)
+                .accessibilityHeading(.h2)
+                .accessibilityLabel(
+                    LocalizedStringKey(
+                        "\(routeType?.typeText(isOnly: true) ?? "") is \(splitStops.collapsedStops.count, specifier: "%ld") stops away from \(splitStops.targetStop.stop.name)"
+                    )
+                )
+                .accessibilityHint(
+                    splitStops.collapsedStops.map(\.stop.name).joined(separator: ", ")
```

Comment:
suggestion: this may be worth confirming with design, but I'd expect this to read "Lists remaining stops" rather than the list of remaining stops. The [hint](https://developer.apple.com/documentation/swiftui/view/accessibilityhint(_:)-3rdgs) is intended to describe to users what would happen if they were to click the button.

Reply:
Yeah that makes sense. I don't think we need to check with design, looking at the Figma I think I mistook the literal text for instructions on what that text should be.